source/security-and-authentication/mongodb-aws-auth.txt

```diff
@@ -39,8 +39,8 @@ connection URI connector property as shown in the following example:
 
    connection.uri=mongodb://<AWS access key id>:<AWS secret access key>@<hostname>:<port>/?authSource=<authentication database>&authMechanism=MONGODB-AWS&authMechanismProperties=AWS_SESSION_TOKEN:<AWS session token>
 
-The preceding example uses the following placeholders which you need to
-replace:
+The preceding example uses the following placeholders which you must
+replace with your own credentials:
```

Comment:
Suggestion:
I think omitting the added text would be more accurate since not all of the placeholders represent credentials.

```suggestion
replace:
```